SPECS/libpostal.spec

```diff
@@ -28,25 +28,28 @@
 # This will make compilation take a *long* time (~1 hour).
 %bcond_with optimize_scanner
 
+# The latest version of libpostal doesn't provide a new version of parser,
+# language_classifier, or libpostal_data, grab them from a previous version
+%global parser_version 1.0.0
+
+
 Name:           libpostal
 Version:        %{rpmbuild_version}
 Release:        %{rpmbuild_release}%{?dist}
 Group:          Applications/Engineering
 Summary:        C library for parsing/normalizing street addresses around the world
 License:        MIT
 URL:            https://github.com/openvenues/libpostal/
-Source0:        https://github.com/openvenues/libpostal/archive/v%{version}/libpostal-%{version}.tar.gz
-Source1:        https://github.com/openvenues/libpostal/releases/download/v%{version}/parser.tar.gz
-Source2:        https://github.com/openvenues/libpostal/releases/download/v%{version}/language_classifier.tar.gz
-Source3:        https://github.com/openvenues/libpostal/releases/download/v%{version}/libpostal_data.tar.gz
+Source0:        https://github.com/openvenues/libpostal/archive/refs/tags/v%{version}.tar.gz
+Source1:        https://github.com/openvenues/libpostal/releases/download/v%{parser_version}/parser.tar.gz
+Source2:        https://github.com/openvenues/libpostal/releases/download/v%{parser_version}/language_classifier.tar.gz
+Source3:        https://github.com/openvenues/libpostal/releases/download/v%{parser_version}/libpostal_data.tar.gz
```

Comment:
Parser, classifier, and data sources weren't included in the latest (2018) release and had to be grabbed from the `v1.0.0` release.  The `https://github.com/openvenues/libpostal/archive/v%{version}/libpostal-%{version}.tar.gz` of the `v1.1` source is corrupted and had to be grabbed from a different location.